src/main/resources/templates/index.html

```diff
@@ -55,9 +51,43 @@
         </p>
     </div>
 
-    <div class="main-grid__output">
-        <label>Output<p class="main-grid__output-text">This is the output</p></label>
-    </div>
+</div>
+
+<div class="main-grid__output main-grid__success">
+    <br>
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24"
```

Comment:
Should we keep these xmlns namespaces?

Reply:
I did a bit of research and I don't think they are necessary, actually. I'll remove them.

Reference the top two comments here for why we don't need it: https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed